tests/PeriodCollectionTest.php

```diff
@@ -138,7 +138,7 @@ public function no_gaps_when_periods_fully_overlap_and_end_excluded()
         $collection = new PeriodCollection(
             Period::make('2018-01-01', '2018-01-05', null, Boundaries::EXCLUDE_END),
             Period::make('2018-01-01', '2018-01-05', null, Boundaries::EXCLUDE_END),
-            Period::make('2018-01-01', '2018-01-05', null, Boundaries::EXCLUDE_END),
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.